### PR TITLE
Get user input for comments and store in Datastore

### PR DESCRIPTION
**Pull request by g-lilian**

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 1]
```diff
@@ -15,28 +15,44 @@
...
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 import java.util.ArrayList;
```
Comment (123cristen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:31]: For import formatting, usually you can just arrange them in alphabetical order, without extra spaces/line breaks.  :)

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java [hunk 2]
```diff
@@ -49,4 +65,30 @@ private String convertToJsonUsingGson(ArrayList<String> listOfStrings) {
     String json = gson.toJson(listOfStrings);
     return json;
   }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    // Get the comment input from the form.
+    String comment = getParameter(request, "text-input", "");
```
Comment (123cristen) [inline portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:72]: Consider checking for an empty string here so you don't store any empty strings

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 1 above)
Comment (g-lilian) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:29]: Ok!

File: portfolio/src/main/java/com/google/sps/servlets/DataServlet.java (hunk 2 above)
Comment (g-lilian) [reply portfolio/src/main/java/com/google/sps/servlets/DataServlet.java:70]: Will do, thanks for the feedback.

**Pull request merged**